osu.Game/Beatmaps/Drawables/Cards/BeatmapCard.cs

```diff
@@ -103,9 +106,19 @@ public static BeatmapCard Create(APIBeatmapSet beatmapSet, BeatmapCardSize size,
             }
         }
 
-        public MenuItem[] ContextMenuItems => new MenuItem[]
+        public MenuItem[] ContextMenuItems
         {
-            new OsuMenuItem(ContextMenuStrings.ViewBeatmap, MenuItemType.Highlighted, Action),
-        };
+            get
+            {
+                var menuItems = new List<MenuItem>();
+
+                if (ButtonContainer.DownloadButton.Enabled.Value)
+                    menuItems.Add(new OsuMenuItem(ContextMenuStrings.DownloadBeatmap, MenuItemType.Highlighted, () => ButtonContainer.DownloadButton.TriggerClick()));
```

Comment:
Never said this question, but what makes menu items highlighted?

Relevant search: https://github.com/search?q=repo%3Appy%2Fosu%20MenuItemType.Highlighted&type=code

I thought the highlighting was to indicate the left-click behavior, but this makes it more inconsistent right now.

The "play" button on beatmap panels in song select was a precedent to these "view" items that I made. Other menus that don't have it highlight something else.